Migration.md

```diff
@@ -645,6 +672,20 @@ ABI was broken for `sdf::Element`, and restored on version 11.2.1.
     `//world/joint` and `//state/insertions/joint` can represent inserted
     `//world/joint` elements.
 
+### Deprecations
+
+1. **joint_state.sdf**:
+    + `//joint_state/angle` is deprecated in favor of `//axis_state/position`
```

Comment:
Just to make it clear that it's still within `joint_state`.
```suggestion
    + `//joint_state/angle` is deprecated in favor of `//joint_state/axis_state/position`
```

Reply:
ok, will fix

Reply:
https://github.com/gazebosim/sdformat/pull/1461/commits/3fa2308158dd6340dd9aa56e820b4f098f9ac688